internal/dashboard/business/obproxy/obproxy.go

```diff
@@ -215,3 +216,31 @@ func DeleteOBProxy(ctx context.Context, ns, name string) (*obproxy.OBProxy, erro
 	}
 	return deleted, nil
 }
+
+func ListOBProxyParameters(ctx context.Context, ns string, name string) ([]obproxy.ConfigItem, error) {
+	items := make([]obproxy.ConfigItem, 0)
+	deploy, err := client.GetClient().ClientSet.AppsV1().Deployments(ns).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		if kubeerrors.IsNotFound(err) {
+			return nil, httpErr.NewNotFound("OBProxy not found")
+		}
+		return nil, httpErr.NewInternal("Failed to get obproxy, err msg: " + err.Error())
+	}
+	odp, err := buildOBProxy(ctx, deploy)
+	if err != nil {
+		return nil, err
+	}
+	for _, pod := range odp.Pods {
+		conn, err := utils.GetOBConnectionByHost(ctx, odp.Namespace, pod.PodIP, "root", "proxysys", odp.ProxySysSecret, 2883)
+		if err != nil {
+			logrus.Infof("Failed to get oceanbase connection by host %s", pod.PodIP)
+			continue
+		}
+		err = conn.QueryList(ctx, &items, "SHOW PROXYCONFIG;")
+		if err != nil {
+			return nil, httpErr.NewInternal("Failed to list obproxy config, err msg: " + err.Error())
+		}
+		return items, nil
```

Comment:
the function needs to list all the parameters of every obproxies

Reply:
By now its target is offering a parameter list. So every pod would return the same list.